contracts/snapshots/DataFeedsCacheGasTest.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This always happens when you run snapshot for data feeds but shouldn't be committed. Can you remove?